packages/neuron-wallet/src/exceptions/address.ts

```diff
@@ -1,18 +1,25 @@
 import i18n from 'locales/i18n'
 
 export class InvalidAddress extends Error {
+  public code = 102
+  public i18n = {
```

Comment:
Does this variable name `i18n` conflict with the import `i18n`?

Reply:
It's the convention used in UI, I'll change the imported name.
But I'm not sure if it's reasonable to include i18n in the custom error class.

Reply:
Maybe [this](https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/src/controllers/api.ts#L389) is a better aspect to handle the i18n

Reply:
Actually I am still not quite sure how this property `i18n` will be used in other places. If it has not yet been referenced in the code, maybe we can remove this property for now and see if there is a better alternative once the code started referencing this variable.

Reply:
Updated in https://github.com/nervosnetwork/neuron/pull/1757/commits/dec1417f15d3e8d47a4364025ce3f7c61861e915

Reply:
The `i18n` property is not used in UI for now, but if some exception happens and UI doesn't handle it, the extra info will be used by i18n in UI automatically.

Reply:
I think I kind of got it now.